pydicom/dataelem.py

```diff
@@ -42,6 +42,42 @@
 ]
 
 
+def empty_value_for_VR(VR, raw=False):
+    """Return the value for an empty element for `VR`.
+
+    The behavior of this property depends on the setting of
+    :attr:`config.use_none_as_empty_value`. If that is set to ``True``,
+    an empty value is always represented by ``None``, otherwise it depends
+    on `VR`. For text VRs (not including VRs that represent a number in
+    textual representation) an empty string is used as empty value
+    representation, for all other VRs, ``None``.
+    Note that this is used only if decoding the element - it is always
+    possible to set the value to another empty value representation,
+    which will be preserved during the element object lifetime.
+
+    Parameters
+    ----------
+    VR : str
+        The VR of the corresponding element.
+
+    raw : bool
+        If ``True``, returns the value for a :class:`RawDataElement`,
+        otherwise for a :class:`DataElement`
+
+    Returns
+    -------
+    str or None
+        The value a data element with `VR` is assigned on decoding
+        if it is empty.
+    """
+    if config.use_none_as_empty_value:
+        return None
+    if VR in ('AE', 'AS', 'CS', 'DA', 'DT', 'LO', 'LT',
+              'PN', 'SH', 'ST', 'TM', 'UC', 'UI', 'UR', 'UT'):
+        return b'' if raw else ''
```

Comment:
I went to see if there is a list like this already defined, and in valuerep.py there is a list called `text_VRs` but much shorter -- it is only those to which specific character set apply.  I don't have an answer, but it would be nice to have more consistency in what we mean by 'text VR'.

Reply:
Ah well, I'm not good with names... I have been thinking about a bunch of constants with VR tuples, such as `numeric_VRs` with subsets `binary_numeric_VRs` and `textual_numeric_VRs` and so on, but we actually don't need them. 
The `text_VRs` tuple is there because I had introduced `convert_text` for these additionally to the existing `convert_string` method (which didn't handle encodings correctly), but I agree it is a bit unclear.
Right now I have no good idea, I'll think about it...